docs/environment.md

```diff
@@ -4,7 +4,86 @@ title: Environment
+        ├── lib
+        ...
+```
+
+These directories are conda environments, and you can use them as such, but you cannot manually edit them, this should always go through the `pixi.toml`.
+Pixi will always make sure the environment is in sync with the `pixi.lock` file.
+If this is not the case all the commands that use the environment will automatically update the environment, e.g. `pixi run`, `pixi shell`.
+
+### Cleaning up
+If you want to clean up the environments, you can simply delete the `.pixi/envs` directory, and pixi will recreate the environments when needed.
+```shell
+rm -rf .pixi/envs
+# or per environment
+rm -rf .pixi/envs/default
+rm -rf .pixi/envs/cuda
+```
+
+## Activation
+As an environment is nothing more than a set of file correctly located in a directory.
+You need to activate the environment to use it.
+In them most simple sense that mean adding the `bin` directory of the environment to the `PATH` variable.
+But there is more to it in a conda environment, as it also sets some environment variables.
+
+To do the activation we have multiple options:
+- Use the `pixi shell` command to open a shell with the environment activated.
+- Use the `pixi shell-hook` command to print the command to activate the environment in your current shell.
+- Use the `pixi run` command to run a command in the environment.
+
+Where the `run` command is special as it runs its own cross-platform shell and has the ability to run tasks.
+More information about tasks can be found in the [tasks documentation](advanced/advanced_tasks.md).
+
+Using the `pixi shell-hook` in pixi you would get the following output:
+```shell
+export PATH="/home/user/development/pixi/.pixi/envs/default/bin:/home/user/.local/bin:/home/user/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/home/user/.pixi/bin"
+export CONDA_PREFIX="/home/user/development/pixi/.pixi/envs/default"
+export PIXI_PROJECT_NAME="pixi"
+export PIXI_PROJECT_ROOT="/home/user/development/pixi"
+export PIXI_PROJECT_VERSION="0.12.0"
+export PIXI_PROJECT_MANIFEST="/home/user/development/pixi/pixi.toml"
+export CONDA_DEFAULT_ENV="pixi"
+export PIXI_ENVIRONMENT_PLATFORMS="osx-64,linux-64,win-64,osx-arm64"
+export PIXI_ENVIRONMENT_NAME="default"
+export PIXI_PROMPT="(pixi) "
+. "/home/user/development/pixi/.pixi/envs/default/etc/conda/activate.d/activate-binutils_linux-64.sh"
+. "/home/user/development/pixi/.pixi/envs/default/etc/conda/activate.d/activate-gcc_linux-64.sh"
+. "/home/user/development/pixi/.pixi/envs/default/etc/conda/activate.d/activate-gfortran_linux-64.sh"
+. "/home/user/development/pixi/.pixi/envs/default/etc/conda/activate.d/activate-gxx_linux-64.sh"
+. "/home/user/development/pixi/.pixi/envs/default/etc/conda/activate.d/libglib_activate.sh"
+. "/home/user/development/pixi/.pixi/envs/default/etc/conda/activate.d/rust.sh"
+```
+It sets the `PATH` and some more environment variables. But more importantly it also runs activation scripts that are presented by the installed packages.
```

Comment:
Do we have a reference as to what activation scripts are or maybe a link to one?

Reply:
Added a link to the glib script